tests/CppUTest/AllTests.cpp

```diff
@@ -46,7 +46,7 @@ int main(int ac, char **av)
         accountant.start();
 #endif
 
-        CommandLineTestRunner::RunAllTests(ac, av); /* cover alternate method */
+        returnValue = CommandLineTestRunner::RunAllTests(ac, av); /* cover alternate method */
```

Comment:
Bugfix so that 'make check' correctly fails if any of the tests fail

Reply:
Please, in a separate PR...